pkg/apis/infinispan/v2alpha1/batch_types.go

```diff
@@ -33,6 +34,8 @@ type BatchStatus struct {
 	Phase BatchPhase `json:"phase"`
 	// Reason indicates the reason for any batch related failures.
 	Reason string `json:"reason,omitempty"`
+	// The UUID of the Infinispan instance that the Batch is associated with
+	InfinispanUUID types.UID `json:"infinispanUUID"`
```

Comment:
We can call it for example `clusterUUID` the same as `cluser` for Infinispan CR name
Also we can use a pointer if we will need to remove it, but probably nothing happens for Batch CR

Reply:
And would be nice to call postfix `UID`, not `UUID`

Reply:
I've renamed it to `ClusterUID`. A pointer seems overkill to me, I think this will always be needed to prevent batch replay.

Reply:
I went for the pointer in the end :smile: